compiler/rustc_smir/src/rustc_smir/mod.rs

```diff
@@ -781,7 +781,7 @@ impl<'tcx> Stable<'tcx> for mir::Terminator<'tcx> {
                 otherwise: targets.otherwise().as_usize(),
             },
             UnwindResume => Terminator::Resume,
-            UnwindTerminate => Terminator::Abort,
+            UnwindTerminate(_) => Terminator::Abort,
```

Comment:
This means the `reason` information is just lost in smir. I wasn't sure what the policy was wrt updating smir when MIR changes.

Reply:
cc @rust-lang/project-stable-mir